deploy check for pmu_name fails for alderlake cpus

Here's my result of running `make deploy`:
```
TASK [intellabs.kafl.kernel : Checking that pmu_name is not broadwell] *********
fatal: [localhost]: FAILED! => changed=false 
  cmd:
  - grep
  - --invert-match
  - broadwell
  - /sys/devices/cpu/caps/pmu_name
  delta: '0:00:00.001088'
  end: '2023-03-02 11:22:22.906029'
  failed_when_result: true
  msg: non-zero return code
  rc: 2
  start: '2023-03-02 11:22:22.904941'
  stderr: 'grep: /sys/devices/cpu/caps/pmu_name: No such file or directory'
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>

PLAY RECAP *********************************************************************
localhost                  : ok=3    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   

make[1]: *** [Makefile:36: deploy] Error 2
make[1]: Leaving directory '/home/user/git/kAFL/deploy'
make: *** [Makefile:27: deploy] Error 2
```

In fact there are two paths:
/sys/devices/cpu_atom/caps/pmu_name
/sys/devices/cpu_core/caps/pmu_name
Both contain "alderlake_hybrid". The CPU is Core i9-13900K

Temporary workaround: delete respective check in the file `deploy/intellabs/kafl/roles/kernel/tasks/pre_tasks.yml`.

--- a/.github/RELEASE.md
+++ b/.github/RELEASE.md
@@ -10,6 +10,7 @@
 
 - include QEMU ROM files in Docker image (#168)
 - push `intellabs/kafl:latest` tag by default (#169)
+- checking `/sys/devices/cpu*/caps/pmu_name` for AlderLake CPUs (#174)
 
 # 📖 Documentation
 

--- a/deploy/intellabs/kafl/roles/kernel/tasks/pre_tasks.yml
+++ b/deploy/intellabs/kafl/roles/kernel/tasks/pre_tasks.yml
@@ -8,10 +8,10 @@
     - hardware_check
 
 - name: Checking that pmu_name is not broadwell
-  ansible.builtin.command: |
-    grep --invert-match broadwell /sys/devices/cpu/caps/pmu_name
-  register: result
-  failed_when: result.rc != 0
+  ansible.builtin.shell: |
+    cat /sys/devices/cpu*/caps/pmu_name
+  register: pmu_name
   changed_when: false
   tags:
     - hardware_check
+  failed_when: pmu_name.stdout.find('broadwell') != -1

